src/Trixi.jl

```diff
@@ -309,6 +311,14 @@ function __init__()
         end
     end
 
+    @static if !isdefined(Base, :get_extension)
```

Comment:
@warisa-r can you add a comment describing the logic behind this (basically re-writing the message you sent me)